### ISSUE TITLE
[UX issue] It's hard to follow the issue title in planner on item details page 

### ISSUE THREAD

**Issue by hrishin** [fabric8io/fabric8-ux#942]
As a user, if I navigate to item detail page it's hard to follow issue title. Also, the box around title it's not comfortable to read.

e.g.
![planner_withbox](https://user-images.githubusercontent.com/28615674/42306796-012b77e4-804e-11e8-9cec-808485c77f5a.png)

Would it be better to change title format? for instance 
![with_bold_lines](https://user-images.githubusercontent.com/28615674/42306925-6f74a586-804e-11e8-8214-3272b1fa3176.png)

**Comment by joshuawilson** [comment]
They are now tracking issues on https://github.com/openshiftio/openshift.io/issues

**Comment by hrishin** [comment]
ack @joshuawilson
